From 1bd77e4a2951abbe2f0e747a860b2d2ce795bd46 Mon Sep 17 00:00:00 2001
From: Carl Edquist <edquist@cs.wisc.edu>
Date: Fri, 2 Jun 2017 11:07:15 -0500
Subject: [PATCH] fix host port test

looks like the meaning of the test was accidentally reversed in aa77601
---
 osg_configure/configure_modules/infoservices.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/osg_configure/configure_modules/infoservices.py b/osg_configure/configure_modules/infoservices.py
index 49fb6b5..8a39afd 100644
--- a/osg_configure/configure_modules/infoservices.py
+++ b/osg_configure/configure_modules/infoservices.py
@@ -298,11 +298,11 @@ class InfoServicesConfiguration(BaseConfiguration):
         host(s) to forward ads to
 
         """
         view_hosts = []
         for host in self.ce_collectors:
-            if ':' in host:
+            if ':' not in host:
                 view_hosts.append("%s:%d" % (host, HTCONDOR_CE_COLLECTOR_PORT))
             else:
                 view_hosts.append(host)
         info_services_file_contents = """\
 # Do not edit - file generated by osg-configure
-- 
2.6.3

